examples/bind/ecommerce.amp.html

```diff
@@ -146,6 +150,9 @@
       - Reintroduce variables in object keys (removed in #7582)
 
     <amp-state [src]="..." on="fetch:AMP.setState({"shirts": {selected: {id: event.response}}})"
+
+    Note that shirt with ID == "0" contains size data (but other IDs don't)
+    because it's the default shirt color at page load.
```

Comment:
Optional: Add "Size data for the remaining indices will be downloaded via XHR when their corresponding options are selected." or something like that.